service/client_state.hh

```diff
@@ -315,6 +328,7 @@ public:
                                       auth::command_desc::type = auth::command_desc::type::OTHER) const;
     future<> has_schema_access(const schema& s, auth::permission p) const;
 
+    future<> update_per_role_params();
```

Comment:
Please add a blurb specifying how timeouts are inherited and conflicts among them resolved.  Also explain why this must be done for all roles rather than a subset of them.

Reply:
done